DigitalLearningSolutions.Data.Tests/Services/UserServiceTests.cs

```diff
@@ -119,7 +120,31 @@ public void GetUsersByUsername_with_admin_id_fetches_associated_delegate_users()
         }
 
         [Test]
-        public void GetAvailableCentres_returns_centres_correctly_ordered()
+        public void GetUsersWithActiveCentres_only_returns_users_with_active_centres()
```

Comment:
I feel like we could take more advantage of moving this into a service as we could also test the case where the admin is returned when the account is for active centre (as currently all tests would pass even if adminUser that's returned is always null)

Reply:
Nevermind on the improved readability question, I realised it was a lot easier than I anticipated for some reason. Made the change to individual parameters rather than a bool array.